docs/manifest-types.md

```diff
@@ -357,13 +357,78 @@ This type of manifest adds a navigation entry into the navigation.xml.
                 "app-catalog-gui": {
                     "label": "Apps",
                     "title": "Apps",
+                    "bundle": "app-catalog-gui",
                     "icon": "fa-archive",
                     "module": "app-catalog-gui",
                     "controller": "index",
                     "action": "index"
                 }
             },
             "after": "users"
+        },
+        {
+            "navigations": {
+                "main": {
+                    "pages": {
+                        "main-nested": {
+                            "bundle": "main",
+                            "controller": "index",
+                            "action": "index",
+                            "visible": "1"
+                        }
+                    }
+                }
+            },
+            "before": "before-main"
+        }
+    ]
+}
+```
+
+### Unwire Navigation Manifest
+
+This type of manifest removes a navigation entry into the navigation.xml.
+
+```json
+{
+    "unwire-navigation": [
+        {
+            "navigations": {
+                "delete": null
+            },
+            "after": "main"
```

Comment:
I think we don't need this line here:

```
"after": "main"
```

`Delete` element will be deleted independent from `after` or `before` value.

Reply:
Thanks 
++